config/locales/en.yml

```diff
@@ -49,20 +49,22 @@ en:
     pgde_with_qts: "PGDE with QTS"
   location_filter:
     errors:
-      no_option: "Please choose an option"
-      unknown_location: "We couldn’t find this location, please check your input and try again"
-      missing_location: "Please enter a postcode, city or town in England"
-      missing_provider: "We couldn’t find this provider, please check your information and try again"
-      blank_provider: "You need to add some information"
-      invalid_provider: "Please enter a minimum of two characters"
```

Comment:
Note: we agreed that "Please enter a minimum of two characters" could be dropped, as the "Enter a real school, university or training provider" message also applies here, so we don't need the separate `invalid_provider` key.